sphinx/source/docs/user_guide/tools.rst

```diff
@@ -486,32 +484,29 @@ in data or screen space. These special fields are listed here:
 Field names that begin with ``@`` are associated with columns in a
 ``ColumnDataSource``. For instance, the field name ``"@price"`` will display
 values from the ``"price"`` column whenever a hover is triggered. If the hover
-is for the 17th glyph, then the hover tooltip will correspondingly display
-the 17th price value.
+is for the 17th glyph, then the hover tooltip will display the 17th price value.
 
-Note that if a column name contains spaces, it must be supplied by
-surrounding it in curly braces, e.g. ``@{adjusted close}`` will display values
+Note that if a column name contains spaces, it must be surrounded by
+curly braces; e.g. ``@{adjusted close}`` will display values
```

Comment:
I think we should semicolons generally (even though I personally like them) and I know @tcmetzger would have all my e.g.'s stricken :) 
```suggestion
curly braces. For example, configuring ``@{adjusted close}`` will display values
```

Reply:
Indeed! 😀

Reply:
BTW, the Google style guide is not completely anti semicolon. They just recommend to use semicolons "judiciously" (https://developers.google.com/style/semicolons). And yes, in this case here a period just works better!